autosubmit/platforms/paramiko_submitter.py

```diff
@@ -83,31 +97,18 @@ def load_platforms(self, asconf, retries=5, auth_password = None, local_auth_pas
                 platforms_used.append(hpc)
         # Traverse used platforms and look for serial_platforms and add them if not already included
         for platform in platforms_used:
-            hpc = asconf.experiment_data.get("PLATFORMS", {}).get(platform, {}).get("SERIAL_PLATFORM", None)
+            hpc = as_conf.experiment_data.get("PLATFORMS", {}).get(platform, {}).get("SERIAL_PLATFORM", None)
             if hpc is not None:
-                platforms_serial_in_paralell[hpc].append(platform)
+                platforms_serial_in_parallel[hpc].append(platform)
                 if hpc not in platforms_used:
                     platforms_used.append(hpc)
 
         platform_data = exp_data.get('PLATFORMS', {})
         # Declare platforms dictionary, key: Platform Name, Value: Platform Object
-        platforms = dict()
+        self.platforms = {}
 
         # Build Local Platform Object
-        local_platform = LocalPlatform(asconf.expid, 'local', exp_data, auth_password=local_auth_password)
-        local_platform.max_wallclock = asconf.get_max_wallclock()
-        local_platform.max_processors = asconf.get_max_processors()
-        local_platform.max_waiting_jobs = asconf.get_max_waiting_jobs()
-        local_platform.total_jobs = asconf.get_total_jobs()
-        local_platform.scratch = os.path.join(
-            BasicConfig.LOCAL_ROOT_DIR, asconf.expid, BasicConfig.LOCAL_TMP_DIR)
-        local_platform.temp_dir = os.path.join(
-            BasicConfig.LOCAL_ROOT_DIR, 'ASlogs')
-        local_platform.root_dir = os.path.join(
-            BasicConfig.LOCAL_ROOT_DIR, local_platform.expid)
-        local_platform.host = 'localhost'
-        # Add object to entry in dictionary
-        platforms['LOCAL'] = local_platform
+        self.load_local_platform(as_conf=as_conf, exp_data=exp_data)
```

Comment:
This code was almost an identical copy of another part of the code in this same file. This new function combines the behaviour of both to reduce code duplication.